### PR TITLE
feat: add tracing.TagDBStatement and tracing.TagDBSqlParameters for gorm

### PR DESCRIPTION
**Pull request by Hair1ossTeenager**
feat: add tracing.TagDBStatement and tracing.TagDBSqlParameters for gorm

### CODE REVIEW

**Comment by wu-sheng** [comment]
You should consider
- Trace SQL parameter should be optional. Because it impacts the performance.
- Update test scenario, https://github.com/apache/skywalking-go/tree/main/test/plugins/scenarios/gorm

**Comment by mrproliu** [comment]
Thanks! Could you also update the [CHANGES.md](https://github.com/apache/skywalking-go/blob/main/CHANGES.md)

**Comment by Hair1ossTeenager** [comment]
> Thanks! Could you also update the [CHANGES.md](https://github.com/apache/skywalking-go/blob/main/CHANGES.md)

done.

File: CHANGES.md [hunk 1]
```diff
@@ -16,6 +16,7 @@ Release Notes.
 * Support higher versions of grpc.
 * Support [go-elasticsearchv8](https://github.com/elastic/go-elasticsearch) database client framework.
 * Support `http.Hijacker` interface for mux plugin.
+* Support `db.statement` and `db.sql.parameters(optional)` for Gorm.
```
Comment (mrproliu) [inline CHANGES.md:19]:
```suggestion
* Support collect statements and parameters in the Gorm plugin. 
```

**Review by mrproliu — APPROVED** [review]
Good! It can be merge after all CI successed.

**Pull request merged**